sbin/get-redisjson

```diff
@@ -76,6 +76,10 @@ else
 	elif [[ $dist == debian ]]; then
 		if [[ $nick == bookworm ]]; then
 			nick="ubuntu22.04"
+		elif [[ $nick == bullseye ]]; then
```

Comment:
What will happen in the case of `bullseye` on x86_64?

Reply:
The RedisJson package for debian bullseye x86_64 will be used.